diff --git a/drivers/tripplite-hid.c b/drivers/tripplite-hid.c
--- a/drivers/tripplite-hid.c
+++ b/drivers/tripplite-hid.c
@@ -82,5 +82,7 @@ static usb_device_id_t tripplite_usb_device_table[] = {
 	{ USB_DEVICE(TRIPPLITE_VENDORID, 0x3016), smart1500lcdt_scale },
 	/* e.g. TrippLite AVR750U (newer unit) */
 	{ USB_DEVICE(TRIPPLITE_VENDORID, 0x3024), smart1500lcdt_scale },
+	/* Tripp Lite BR1500LCDT; validated with USB product ID 3028 */
+	{ USB_DEVICE(TRIPPLITE_VENDORID, 0x3028), smart1500lcdt_scale },
 	/* e.g. TrippLite SmartOnline SU1500RTXL2UA (older unit?) */
 	{ USB_DEVICE(TRIPPLITE_VENDORID, 0x4001), battery_scale_1dot0 },
